### ISSUE TITLE
BEE2 missing msvcp140.dll from Visual C++ Redistributable

### ISSUE THREAD

**Issue by Laugier2** [BEEmod/BEE2.4#1555]
**Hello, I downloaded Beemod installed the packages run for the first time and error. . .**
BEE2.4 Version 4.39.2

**BEE2 log**
```
[INFO] utils.setup_localisations(): Language: 'fr_FR'
[DEBUG] utils.setup_localisations(): Language codes: ['fr_FR.ISO8859-1', 'fr_FR', 'fr.ISO8859-1', 'fr']
[INFO] BEE2_launch.<module>(): Arguments: ['C:\\Program Files (x86)\\Steam\\steamapps\\common\\Portal 2\\portal2\\beeemod2.4\\BEE2.exe']
[INFO] BEE2_launch.<module>(): Running "bee2", version 4.39.1:
[WARNING] BEE2_config.load(): Config "C:\Users\niten\AppData\Roaming\BEEMOD2\config\packages.cfg" not found! Using defaults...
[ERROR] (unknown file).(unknown function)(): Uncaught Exception:
Traceback (most recent call last):
  File "BEE2_launch.pyw", line 49, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\BEE2.py", line 5, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\gameMan.py", line 31, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\optionWindow.py", line 17, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\contextWin.py", line 22, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\itemconfig.py", line 14, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\UI.py", line 25, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\item_search.py", line 8, in <module>
ImportError: DLL load failed while importing marisa_trie: Le module spécifié est introuvable.
```

**BEE2.ERROR**
```
[WARNING] BEE2_config.load(): Config "C:\Users\niten\AppData\Roaming\BEEMOD2\config\packages.cfg" not found! Using defaults...
[ERROR] (unknown file).(unknown function)(): Uncaught Exception:
Traceback (most recent call last):
  File "BEE2_launch.pyw", line 49, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\BEE2.py", line 5, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\gameMan.py", line 31, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\optionWindow.py", line 17, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\contextWin.py", line 22, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\itemconfig.py", line 14, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\UI.py", line 25, in <module>
  File "PyInstaller\loader\pyimod03_importers.py", line 531, in exec_module
  File "app\item_search.py", line 8, in <module>
ImportError: DLL load failed while importing marisa_trie: Le module spécifié est introuvable.
```

**Comment by KaziPlays** [comment]
I have the same error
![image](https://user-images.githubusercontent.com/72144613/116911115-cccd4b00-ac1c-11eb-8f7a-96c00df4bc17.png)

**Comment by KaziPlays** [comment]
"The specified module could not be found" That is what it says.

**Comment by KaziPlays** [comment]
I'm running sfc /scannow on Admin command prompt, if I succeed I'll edit this.

**Comment by KaziPlays** [comment]
I recommend you add "ImportError: DLL load failed while importing marisa_trie: The specified module could not be found." on the title.

**Comment by KaziPlays** [comment]
I solved it by installing Visual C++ Redistributable, but a RIFF ID error appeared, Even though I'm in windows 10 64 bits with 64-bit Beemod

**Comment by DrMeepster** [comment]
It's weird that installing Visual C++ Redistributable fixes this because beemod comes with the redistributable dlls

**Comment by DrMeepster** [comment]
`marisa_trie.cp39-win_amd64.pyd` seems to rely on `msvcp140.dll` which is not included with beemod. I think distributing that with beemod would fix this issue

**Comment by LautaroL20** [comment]
If the problem here was the lack of Visual C++ Redistributable, then let´s turn this issue into an enhancement as well.

**Comment by Luke18033** [comment]
No, the bug is that you shouldn't need to manually install that.

**Comment by DrMeepster** [comment]
Wasn't this solved by d2a0218991a474e001e9e819f6ea93199f962d94?

**Comment by Luke18033** [comment]
Actually yeah probably. Let me know if this still happens and I'll reopen it.
